core/Pos/Arbitrary/Core.hs

```diff
@@ -285,6 +286,30 @@ newtype SafeWord = SafeWord
 instance Arbitrary SafeWord where
     arbitrary = SafeWord . Types.getCoinPortion <$> arbitrary
 
+-- | A wrapper over 'Int'. When converted to 'EpochOrSlot' via 'fromEnum', using this
```

Comment:
It used to be a wrapper over `Int'` for about a minute before I realized it had to be a `EpochOrSlot`.